docs/ui/auth/fragments/angular/custom-form-fields.md

```diff
@@ -15,21 +15,21 @@ export class AppComponent {
     this.formFields = [
       {
         type: "email",
-        label: "Custom email Label",
-        placeholder: "custom email placeholder",
-        required: true,
+        label: "Custom Email Label",
+        placeholder: "Custom email placeholder",
+        inputProps: { required: true, autocomplete: "username" },
       },
       {
         type: "password",
         label: "Custom Password Label",
-        placeholder: "custom password placeholder",
-        required: true,
+        placeholder: "Custom password placeholder",
+        inputProps: { required: true, autocomplete: "new-password" },
       },
       {
         type: "phone_number",
         label: "Custom Phone Label",
-        placeholder: "custom Phone placeholder",
-        required: false,
+        placeholder: "Custom phone placeholder",
+        inputProps: { required: true },
```

Comment:
Why is this now required (was `required: false`)?

Reply:
Not gonna lie, this was a copy paste error :D Let me revert this back to false.

Reply:
Actually, we don't even need `inputProps` if we do not want `required`. I'll delete these lines.